package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nats",
-  "version": "2.17.0",
+  "version": "2.17.1-1",
```

Comment:
Why the dash?

Reply:
Because that is a test version - will only make it to nats@next - that is a qualifier.